lib/tether/ops_linux.go

```diff
@@ -73,6 +73,12 @@ const (
 	nfsFileSystemType  = "nfs"
 	ext4FileSystemType = "ext4"
 	bridgeTableNumber  = 201
+
+	defaultNOFILE = 1024 * 1024
```

Comment:
please add brief comments above all of these constant values to describe what they do and why they were picked :)